Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Menu/MenuPage.xaml

```diff
@@ -127,11 +123,12 @@
                     <MenuFlyoutItem Text="Grid" />
                 </MenuFlyoutSubItem>
             </controls:MenuItem>
+
         </controls:Menu>
 
         <StackPanel Grid.Row="1">
             <TextBlock Text="Click Alt to set focus on Classic Menu" />
-            <TextBlock Text="Any menu shortcut is Alt + (The first character of the menu name), ex to open File menu click Alt+F" />
+            <TextBlock Text="To know any menu shortcut click Alt + (The underlined character in case underline ^ is used), ex to open File menu click Alt+F" />
```

Comment:
This line is unclear, could you update it to something along the lines of: Hold Alt to underline shortcut character. In code, ^ is used before the character you want highlighted"